app/locales/en/translations.js

```diff
@@ -1263,6 +1249,23 @@ export default {
     }
   },
   models: {
+    imaging: {
+      labels: {
+        requestedDate: 'Date Requested',
+        imagingDate: 'Date Completed',
+        requestedBy: 'Requested By',
+        radiologist: 'Radiologists',
+        patient: 'Patient',
+        imagingType: 'Imaging Type',
+        result: 'Result',
+        notes: 'Notes',
+        visit: 'Visit',
+        status: 'Status'
+      },
+      placeholders: {
```

Comment:
@billybonks I'm not sure about calling this a placeholder.  It is a valid option to select, not a temporary placeholder value.  When you select/leave this option a new visit is created, so there is action firing based on its selection/non selection.